src/aggregator/aggregator/election_mgr.go

```diff
@@ -651,6 +655,12 @@ func (mgr *electionManager) campaignIsEnabled() (bool, error) {
 		hasNotCutoff := nowNanos < shard.CutoffNanos()-int64(mgr.shardCutoffCheckOffset)
 		if hasCutover && hasNotCutoff {
 			mgr.metrics.campaignCheckHasActiveShards.Inc(1)
+			if mgr.ElectionState() == LeaderState {
+				mgr.metrics.leadersWithActiveShards.Update(float64(1))
```

Comment:
I'm kinda confused about this metric. So if the host is the leader for ANY shards, you want to set this entire gauge to 1? Also the name is a little confusing consider the gauge value will only apply to a single host

Reply:
The PR tries to emit a metric for both leaders and followers that have active shards so that alerts can be configured to only check for a mismatch of leaders and non-leaders that do not include newly added hosts that are bootstrapping. 

I agree that gauge is a bit awkward but wasn't sure how best to express this type of metric. 

Should I just use the query that is within the description of the PR as a way to calculate this type of metric?